### PR TITLE
Description for everyone else

### PR DESCRIPTION
**Pull request by lamagnifica**
This PR adds a description for the EdX Video ID field, which everyone will see, to the Building and Running an edX Course guide.

@nasthagiri, @frrrances, @WatsonEmily , @lou-wang, @srpearce 
@mhoeber, @catong FYI

### CODE REVIEW

File: en_us/course_authors/source/creating_content/create_video.rst [hunk 1]
```diff
@@ -309,6 +309,9 @@ The following options appear on the **Advanced** tab in the video component.
         students by using the **Upload Handout** field. For more information,
         see :ref:`Additional Transcripts`.
 
+    * - **EdX Video ID**
+      - This optional field is used only by course teams that are working with
```
Comment (srpearce) [inline en_us/course_authors/source/creating_content/create_video.rst]: Just to make this more consistent with the other items, I'd change slightly: "An optional field used only by course teams..."

**Comment by srpearce** [comment]
Nit; otherwise :+1:

**Comment by catong** [comment]
:+1:

**Comment by WatsonEmily** [comment]
:+1:

**Comment by lou-wang** [comment]
:+1:

**Comment by nasthagiri** [comment]
:+1:

**Pull request merged**